statsmodels/genmod/cov_struct.py

```diff
@@ -927,9 +927,22 @@ class Equivalence(CovStruct):
       one triangle of each covariance matrix should be included.
       Positions where j1 and j2 have the same value are variance
       parameters.
+    labels : array-like
+      An array of labels such that every distinct pair of labels
+      defines an equivalence class.  Either `labels` or `pairs` must
+      be provided.  When the two labels in a pair are equal two
+      equivalence classes are defined: one for the diagonal elements
+      (corresponding to variances) and one for the off-diagonal
+      elements (corresponding to covariances).
+    as_cor : boolean
```

Comment:
since it will be only binary, a boolean would be better?
If it's not a very frequently used option, then we could use `return_correlation` or `return_cov=False`.
One problem is that `cor` is not very common as an abbreviation, ,maybe it should be `corr`. (maybe the only similar case is `nearest_corr` or something like that in correlation_tools.py)